suse/x86_64/suse-leap-15.3/root/etc/motd

```diff
@@ -0,0 +1,7 @@
+This is an openSUSE Leap 15.2 Appliance System.
```

Comment:
@schaefi & @db-at-work 
I think we have an inadvertent "15.2" here when likely "15.3" was intended.
Sorry for the late spot of this as I know it's already been merged.

Reply:
new PR generated for this. well spotted. thanks.